server/src/main/java/org/elasticsearch/action/admin/cluster/state/ClusterStateResponse.java

```diff
@@ -75,6 +78,14 @@ public ByteSizeValue getTotalCompressedSize() {
         return totalCompressedSize;
     }
 
+    /**
+     * Returns whether waiting for a cluster state wait a metadata version equal or higher than the specified
+     * metadata version timed out.
+     */
+    public boolean isTimedOut() {
```

Comment:
The Javadocs are clear but I think the name could be clearer and reflect the name of the parameter on the request: `isWaitForTimedOut`